AllASPFuncs.py

```diff
@@ -1449,11 +1450,21 @@ def calc_mags_and_err(flux, sigma_flux, band, zp = None):
             'Y106': 302.275,
             'Z087': 101.7}
 
+    flux = np.atleast_1d(flux)
+    sigma_flux = np.atleast_1d(sigma_flux)
```

Comment:
Why does `calc_mags_and_err` change scalar input to an array?  I would find this behavior surprising, particularly because it's not documented.